runtime/libc/wasi_sdk_backing.c

```diff
@@ -181,6 +181,8 @@ enum wasi_rights {
     WASI_RIGHT_SOCK_SHUTDOWN            = 0x0000000010000000ULL,
 };
 
+typedef uint16_t wasi_errno_t;
```

Comment:
OMG, I can't thank you enough for fixing this up. Was a pain in the ass to cross reference all of these to the standard. Thank you!